ncm-modprobe/src/main/perl/modprobe.pm

```diff
@@ -57,7 +58,9 @@ sub mkinitrd
     foreach my $i (@$dir) {
         if ($i =~ m{^System\.map\-(2\.6\..*)$}) {
```

Comment:
not relevant for kernels more recent than 2.6? or is this code that means older than 2.4?